### ISSUE TITLE
Minio Service Uptime & Minio Server Uptime aren't different

### ISSUE THREAD

**Issue by abhinav-chittora** [joe-pll/minio-exporter#8]
Currently, minio_server_uptime metrics show minio_uptime data. However, there are chances, that service uptime (minio_uptime) can be different than server uptime (minio_server_uptime). 

https://github.com/joe-pll/minio-exporter/blob/master/minio_exporter.go#L168 issue seems at this line. 

Environment: 
OS Version: Centos 7
MInio Version: 2018-01-18T20:33:21Z

**Comment by joe-pll** [comment]
The values are coming from two different data structures of Minio.
The minio_uptime comes from ServiceStatus.Uptime, while the minio_server_uptime comes from the ServerInfo structure.
I tested on my own and stopped one of the servers. The server uptime is different from the ServiceStatus uptime. If you have one service only these values are the same.

**Comment by abhinav-chittora** [comment]
I am using minio on single system and minio_exporter returns the value for service uptime & server uptime. Is there any way to debug outputs?

**Comment by joe-pll** [comment]
Yes, it is normal to return both values. In the Minio API these values are exposed differently. But in case of a single server are the same. What do you mean with "debug outputs"?

**Comment by abhinav-chittora** [comment]
The problem is both parameters return the same values however, system uptime and Service uptime are different. 

Debug output = verbose mode.
